changelog.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+### 0.3
```

Comment:
Previously this project followed `X.Y.Z` versions, and it's very common. Why switched to shorter format?